app/races/stats/[game]/category-stats-list.tsx

```diff
@@ -82,14 +85,19 @@ export const CategoryStatsBody = ({ stats }: { stats: CategoryStats }) => {
                 <div>
                     Total Time:{" "}
                     <b>
-                        <DurationToFormatted duration={stats.totalRaceTime} />
+                        <DurationToFormatted
+                            duration={stats.totalRaceTime}
+                            padded={true}
+                        />
                     </b>
                 </div>
                 <div>
-                    Races Joined: <b>{stats.totalParticipations}</b>
+                    Races Joined:{" "}
+                    <b>{DigitGrouping(stats.totalParticipations)}</b>
                 </div>
                 <div>
-                    Races Finished: <b>{stats.totalFinishedParticipations}</b>
+                    Races Finished:{" "}
```

Comment:
Can we replace these `{" "}` with the html entity for a non-breaking space instead

```suggestion
                    Races Finished:&nbsp;
```